relayer-cli/src/testnet/arbSepToSepRelayer.ts

```
@@ -0,0 +1,38 @@
+  process.on(signal, async () => {
+    console.log("exit");
+    const lock_file_name = "./src/state/" + chain_id + ".pid";
+    if (fs.existsSync(lock_file_name)) {
+      fs.unlinkSync(lock_file_name);
+    }
+    process.exit(0);
+  })
+);
+
+(async () => {
+  while (1) {
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)